src/geometry/manifold/charted_space.lean

```diff
@@ -6,62 +6,85 @@ Authors: Sébastien Gouëzel
 import topology.local_homeomorph
 
 /-!
-# Manifolds
-
-A manifold is a topological space M locally modelled on a model space H, i.e., the manifold is
-covered by open subsets on which there are local homeomorphisms (the charts) going to H. If the
-changes of charts satisfy some additional property (for instance if they are smooth), then M
-inherits additional structure (it makes sense to talk about smooth manifolds). There are therefore
-two different ingredients in a manifold:
+# Charted spaces
+
+A smooth manifold is a topological space `M` locally modelled on a euclidean space (or a euclidean
+half-space for manifolds with boundaries, or an infinite dimensional vector space for more general
+notions of manifolds), i.e., the manifold is covered by open subsets on which there are local
+homeomorphisms (the charts) going to a model space`H`, and the changes of charts should be smooth
+maps.
+
+In this file, we introduce a general framework describing these notions, where the model space is an
+arbitrary topological space. We avoid the word *manifold*, which should be reserved for the
+situation where there are also notions of smoothness and dimension, and use the terminology
+"charted space" instead.
```

Comment:
I haven't studied these files yet, so maybe this is too naïve. Topological manifolds don't need any notion of smoothness, right? (Are they defined somewhere?)

Reply:
You're right. I rephrased things.